frontend/public/components/storage/create-pvc.tsx

```diff
@@ -19,19 +19,37 @@ import { PersistentVolumeClaimModel } from '../../models/index';
 
 const NameValueEditorComponent = (props) => <AsyncComponent loader={() => import('../utils/name-value-editor').then(c => c.NameValueEditor)} {...props} />;
 
+//See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes for more details
+const provisionerAccessModeMapping = {
+  'kubernetes.io/no-provisioner': ['ReadWriteOnce'],
+  'kubernetes.io/aws-ebs' : ['ReadWriteOnce'],
+  'kubernetes.io/gce-pd' : ['ReadWriteOnce', 'ReadOnlyMany'],
+  'kubernetes.io/glusterfs' : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'],
+  'kubernetes.io/cinder' : ['ReadWriteOnce'],
+  'kubernetes.io/azure-file' : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'],
+  'kubernetes.io/azure-disk' : ['ReadWriteOnce'],
+  'kubernetes.io/quobyte' : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'],
+  'kubernetes.io/rbd' : ['ReadWriteOnce', 'ReadOnlyMany'],
+  'kubernetes.io/vsphere-volume' : ['ReadWriteOnce', 'ReadWriteMany'],
+  'kubernetes.io/portworx-volume' : ['ReadWriteOnce', 'ReadWriteMany'],
+  'kubernetes.io/scaleio' : ['ReadWriteOnce', 'ReadOnlyMany'],
+  'kubernetes.io/storageos' : ['ReadWriteOnce'],
+};
+
 // This form is done a little odd since it is used in both its own page and as
 // a sub form inside the attach storage page.
 export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVCFormState> {
   state = {
     storageClass: '',
-    storageClassAccessMode: '',
+    storageClassAccessModes: ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'],
     pvcName: '',
     accessMode: 'ReadWriteOnce',
     requestSizeValue: '',
     requestSizeUnit: 'Gi',
     disableForm: false,
     useSelector: false,
     nameValuePairs: [['', '']],
+    message: 'Permissions to the mounted drive.',
```

Comment:
This needs a more specific name like `accessModesHelp`

Reply:
ack